### DISCUSSION TITLE
Changing Repository Name

### DISCUSSION THREAD

**Discussion by pratiksoni1402** [community/community#72621] [Repositories] [answered]
### Select Topic Area

Question

### Body

I want to know that if i change the repository name will my contribution in that repo also gets deleted or they will stay as it is.

**Comment by andrej-kocijan** [comment, marked as answer]
Everything should be fine. Just do not reuse the old name on a new repository. You can read more about renaming repositories [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository)

**Reply by pratiksoni1402** [reply]
Thanks

**Comment by tuves** [comment]
Welcome to the GitHub Community, @pratiksoni1402, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
